templates/pages/core/get-started/developer-setup.md

```diff
@@ -54,3 +64,183 @@ It requires you to have ran `snapcraft` on your project folder at least once, so
 * Run `snap try` in your project folder
 
 The `snap enable` and `disable` commands will let you stop and restart your snap after you changed its code, if for example, it is providing a service that runs when the snap is installed.
+
+## Installing additional snaps
+
+When developing on your device, it can be useful to add the following snaps to enable specific features.
+
+### pulseaudio
+
+Available on armhf, i386, amd64, arm64.
+
+PulseAudio is a sound server for POSIX and Win32 systems. To record or playback audio files, you need to install the pulseaudio snap.
+
+#### Installing pulseaudio
+
+    sudo snap install pulseaudio
+
+#### Using pulseaudio
+
+Note that when executing the following commands, a warning message can be printed ("Failed to create secure directory..."), it can be ignored.
+
+* List all audio modules and devices:
+
+        sudo pulseaudio.pactl list
+
+* List output ports (sinks):
+
+        sudo pulseaudio.pactl list sinks
+
+* List input ports (sources):
+
+        sudo pulseaudio.pactl list sources
+
+* List audio cards:
+
+        sudo pulseaudio.pactl list cards
+
+* Set output and input profile
+    * Analog output and analog input:
+
+            sudo pulseaudio.pactl set-card-profile 0 output:analog-stereo+input:analog-stereo
+
+    * Analog output and analog input:
+
+            sudo pulseaudio.pactl set-card-profile 0 output:hdmi-stereo+input:analog-stereo
+
+    Note: the argument `0` is sequence and `output:analog-stereo+input:analog-stereo` and `output:hdmi-stereo+input:analog-stereo` are profiles. They are listed in card information, check the ‘List audio cards’ command above.
+
+* Mute or unmute
+
+    * Mute an output port:
+
+            sudo pulseaudio.pactl set-sink-mute 2 1
+
+        `2` is the sequence of output port which is listed by ‘List output ports (sinks)’. `1` means mute.
```

Comment:
Shouldn't this be aligned with the "Mute" above? It's 2 spaces in. same goes for many others like this. It's really the same point as I made above.